src/auth/screens/events.screen.js

```diff
@@ -149,7 +149,7 @@ class Events extends Component {
       case 'MemberEvent':
         return `${userEvent.payload.action}`;
       case 'PublicEvent':
-        return translate('auth.events.issuesEvent', language);
+        return translate('auth.events.publicEvent', language);
```

Comment:
Ok, no need to keep my PR then. We will come up with a cleaner solution later.
Nice work !